### PR TITLE
fix: lines_longer_than_80_chars errors + clean-up

### PR DESCRIPTION
**Pull request by HelenGezahegn**
**Changes**
- fixed out-of-sync podfile
- deleted `mentorApp` folder
- added .last_build_id to gitignore
- resolved existing effective dart violations in codebase
- fixed any violations of `lines_longer_than_80_chars`

**UI**
> No visible changes

**Note**
As I regenerated the Podfile, please test this out in your local dev environment to ensure there are no issues.

### CODE REVIEW

File: lib/navigation/app_controller.dart [hunk 1]
```diff
@@ -22,41 +20,42 @@ class AppController extends StatefulWidget {
...
-  double navBarIconSize = 30; 
+  int pageIndex = 1;
+  double navBarIconSize = 30;
   final List<Widget> _widgets = [UserProfile(), Pairings(), Chat()];
 
   tapped(int tappedIndex) {
-    setState(() {pageIndex = tappedIndex;});
+    setState(() {
+      pageIndex = tappedIndex;
+    });
```
Comment (xiangxinyue) [inline lib/navigation/app_controller.dart:32]: Thanks for fixing the index!

File: lib/ftu/signuplogin.dart [hunk 2]
```diff
@@ -53,11 +52,11 @@ class _LoginSignupPageState extends State<LoginSignupPage> {
         if (userId.length > 0 && userId != null && _isLoginForm) {
           widget.loginCallback();
         }
-      } catch (e) {
+      } on Exception catch (e) {
```
Comment (xiangxinyue) [inline lib/ftu/signuplogin.dart:55]: Thanks for solving the warning here!

File: lib/navigation/app_controller.dart [hunk 3]
```diff
@@ -22,41 +20,42 @@ class AppController extends StatefulWidget {
...
+              height: appNavBarHeight,
+              index: 1,
+              onTap: tapped,
+              items: <Widget>[
+                Icon(Icons.person, size: navBarIconSize),
+                Icon(Icons.home, size: navBarIconSize),
+                Icon(Icons.chat_bubble, size: navBarIconSize),
+              ],
+              animationDuration: Duration(milliseconds: 200),
+              animationCurve: Curves.bounceInOut),
```
Comment (xiangxinyue) [inline lib/navigation/app_controller.dart:51]: Looks better!

File: lib/profile/sections/section.dart [hunk 4]
```diff
@@ -1,104 +1,78 @@
...
     return Container(
-      padding: EdgeInsets.only(bottom: 10.0),
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: children
-      )
-    );
+        padding: EdgeInsets.only(bottom: 10.0),
+        child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start, children: children));
```
Comment (xiangxinyue) [inline lib/profile/sections/section.dart:76]: I would put it  two lines:

child: Column(
        crossAxisAlignment: CrossAxisAlignment.start,
        children: children
      )

File: lib/chat/chat_messenger.dart [hunk 5]
```diff
@@ -61,66 +50,60 @@ class ChatMessengerState extends State<ChatMessenger> {
...
-          IconButton(
-            icon: Icon(Icons.send),
-            iconSize: 25.0,
-            color: Theme.of(context).primaryColor,
-            onPressed: () => submitMessage(textInputController.value.text),
-          ),
-        ]
-      )
-    );
+            ]));
```
Comment (eivenlour) [inline lib/chat/chat_messenger.dart:106]: Indentation

File: lib/profile/sections/section.dart [hunk 6]
```diff
@@ -1,104 +1,78 @@
...
-        children: children
-      )
-    );
+        width: MediaQuery.of(context).size.width,
+        padding: EdgeInsets.only(bottom: 10.0),
+        margin: EdgeInsets.only(bottom: 10.0),
+        decoration: BoxDecoration(
+            border: Border(bottom: BorderSide(color: Colors.black12))),
+        child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start, children: children));
```
Comment (xiangxinyue) [inline lib/profile/sections/section.dart:63]: Same here, I would make it two lines

child: Column(
        crossAxisAlignment: CrossAxisAlignment.start,
        children: children
      )

File: lib/chat/message.dart [hunk 7]
```diff
@@ -4,30 +4,28 @@ import 'package:flutter/material.dart';
...
+                ? EdgeInsets.fromLTRB(50.0, 10.0, 10.0, 10.0)
+                : EdgeInsets.fromLTRB(10.0, 10.0, 50.0, 10.0),
+            padding: EdgeInsets.all(10.0),
+            child: Text(message,
+                style: GoogleFonts.muli(
+                    textStyle: TextStyle(
+                  color: Colors.white,
+                  letterSpacing: .5,
+                  fontSize: 14.0,
+                )))));
```
Comment (eivenlour) [inline lib/chat/message.dart:29]: damn, is this how we're supposed to do it? :sob: so sad, i actually felt like the other way was easier to read  😢

File: lib/profile/sections/section.dart (hunk 4 above)
Comment (eivenlour) [reply lib/profile/sections/section.dart:76]: @xiangxinyue I think this is part of the fix... ?

File: lib/profile/sections/section.dart (hunk 6 above)
Comment (eivenlour) [reply lib/profile/sections/section.dart:63]: @xiangxinyue same here! I think this is part of the fix Helen is making.

File: lib/ftu/signuplogin.dart (hunk 2 above)
Comment (HelenGezahegn) [reply lib/ftu/signuplogin.dart:55]: No problem! 😘

File: lib/navigation/app_controller.dart (hunk 1 above)
Comment (HelenGezahegn) [reply lib/navigation/app_controller.dart:32]: It was the automatic formatting! 🤪

File: lib/chat/message.dart (hunk 7 above)
Comment (HelenGezahegn) [reply lib/chat/message.dart:29]: Yeah, I didn't even notice how crazy it looked as I just assumed the auto-formatter would make it look better... 😭

File: lib/profile/sections/section.dart (hunk 4 above)
Comment (xiangxinyue) [reply lib/profile/sections/section.dart:76]: @eivenlour  u mean automatic formatter?

File: lib/profile/sections/section.dart (hunk 6 above)
Comment (xiangxinyue) [reply lib/profile/sections/section.dart:63]: @eivenlour oh great

**Comment by HelenGezahegn** [comment]
> LOOKS SO MUCH CLEANER!! THANK YOU FOR DOING THIS!!!

AWWWWW, ILU LOL

**Pull request merged**